prac_05/state_names.py

```diff
@@ -0,0 +1,18 @@
+CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
+                "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
+state_lists = []
+print(CODE_TO_NAME)
+state_code = input("Enter short state: ").upper()
+while state_code != "":
+    if state_code in CODE_TO_NAME:
+        print(f"{state_code:<3} is {CODE_TO_NAME[state_code]}")
+        state_lists.append(state_code)
+    else:
+        print("Invalid short state")
```

Comment:
I think there should be a input statement after the else part after invalid short state.